docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Conversion/Hash.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public static class Hash
+    {
```

Comment:
Please add xml string --- This example demonstrates hashing of string and integer types.